### PR TITLE
Fix: quotes

### PR DESCRIPTION
**Pull request by Gootector**
Fix: quotes

Greetings,
Gootector

Resolves: #NNNNN 

- [x] I signed the [CLA](https://musescore.org/en/cla)
- [x] The title of the PR describes the problem it addresses
- [x] Each commit's message describes its purpose and effects, and references the issue it resolves
- [x] If changes are extensive, there is a sequence of easily reviewable commits
- [x] The code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] There are no unnecessary changes
- [x] The code compiles and runs on my machine, preferably after each commit individually
- [x] I created a unit test or vtest to verify the changes I made (if applicable)

### CODE REVIEW

**Review by shoogle — APPROVED** [review]

File: src/notation/internal/notationuiactions.cpp [hunk 1]
```diff
@@ -215,7 +215,7 @@ const UiActionList NotationUiActions::m_actions = {
     UiAction("toggle-insert-mode",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_ANY,
-             TranslatableString("action", "Toggle 'insert mode'"),
+             TranslatableString("action", "Toggle ‘insert mode’"),
```
Comment (Jojo-Schmitz) [inline src/notation/internal/notationuiactions.cpp:218]: Shouldn't it be `Insert mode`, capital i?

File: src/notation/internal/notationuiactions.cpp [hunk 2]
```diff
@@ -1882,13 +1882,13 @@ const UiActionList NotationUiActions::m_actions = {
     UiAction("toggle-autoplace",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_ANY,
-             TranslatableString("action", "Toggle 'automatic placement' for selected elements"),
+             TranslatableString("action", "Toggle ‘automatic placement’ for selected elements"),
```
Comment (Jojo-Schmitz) [inline src/notation/internal/notationuiactions.cpp:1885]: and capital 'a'  here

File: src/notation/internal/notationuiactions.cpp [hunk 3]
```diff
@@ -1882,13 +1882,13 @@ const UiActionList NotationUiActions::m_actions = {
...
              TranslatableString("action", "Toggle automatic placement for selected elements")
              ),
     UiAction("autoplace-enabled",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_NOTATION_OPENED,
-             TranslatableString("action", "Toggle 'automatic placement' (whole score)"),
+             TranslatableString("action", "Toggle ‘automatic placement’ (whole score)"),
```
Comment (Jojo-Schmitz) [inline src/notation/internal/notationuiactions.cpp:1891]: Here too. Not sure about the 'w'?

**Comment by Jojo-Schmitz** [comment]
Still ingnores alls the "Reset 'xxx' to default value(s)". But those are for screen readers only, so I'm not sure whether it matters, @shoogle ?

**Comment by Gootector** [comment]
@cbjeukendrup Please review and merge (if it's okay).

File: src/notation/internal/notationuiactions.cpp [hunk 4]
```diff
@@ -1882,13 +1882,13 @@ const UiActionList NotationUiActions::m_actions = {
     UiAction("toggle-autoplace",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_ANY,
-             TranslatableString("action", "Toggle 'automatic placement' for selected elements"),
-             TranslatableString("action", "Toggle automatic placement for selected elements")
+             TranslatableString("action", "Toggle ‘Automatic placement’ for selected elements"),
+             TranslatableString("action", "Toggle ‘automatic placement’ for selected elements")
```
Comment (cbjeukendrup) [inline src/notation/internal/notationuiactions.cpp:1886]: Is it intentional that the second one is without uppercase A?

Comment (Gootector) [reply src/notation/internal/notationuiactions.cpp:1886]: "Toggle ‘automatic placement’ for selected elements" (line 1886) and "Toggle ‘automatic placement’ (whole score)" (line 1892).

Comment (Gootector) [reply src/notation/internal/notationuiactions.cpp:1886]: Should be uppercase A?

Comment (Jojo-Schmitz) [reply src/notation/internal/notationuiactions.cpp:1886]: IMHO yes

Comment (Gootector) [reply src/notation/internal/notationuiactions.cpp]: Done.

**Comment by cbjeukendrup** [comment]
Looks fine to me now; @shoogle please check if you agree too :)

**Comment by Gootector** [comment]
> Looks fine to me now; @shoogle please check if you agree too :)

What about "Toggle ‘rhythmic slash notation’" (line 452)?

**Comment by Jojo-Schmitz** [comment]
> What about "Toggle ‘rhythmic slash notation’" (line 452)?

Capital R

**Comment by Gootector** [comment]
OK. Done.

**Comment by Gootector** [comment]
Edit -> Preferences -> Shortcuts:

Toggle ‘Automatic placement’ (whole score)
Toggle ‘Automatic placement’ for selected elements
Toggle ‘Loop playback’
Toggle ‘Rhythmic slash notation’

IMO is OK. Please merge.

**Review by Jojo-Schmitz — APPROVED** [review]

**Comment by shoogle** [comment]
I agree with all the changes outside of `notationuiactions.cpp`.

Within `notationuiactions.cpp`, I don't think we really need quotes around `Toggle ‘attribute’` unless `attribute` starts with a verb (e.g. `Toggle ‘hide empty staves’`). Also, I notice that `attribute` starts with a lowercase letter more often that with an uppercase letter in that file, and I believe this is intentional.

However, I have a new proposal that gets rid of quotes altogether in that file:

<details><summary><b>Non-note input options:</b></summary>

```diff
diff --git a/src/notation/internal/notationuiactions.cpp b/src/notation/internal/notationuiactions.cpp
index b417f996a1..3562f94477 100644
--- a/src/notation/internal/notationuiactions.cpp
+++ b/src/notation/internal/notationuiactions.cpp
@@ -449,7 +443,7 @@ const UiActionList NotationUiActions::m_actions = {
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_NOTATION_OPENED,
              TranslatableString("action", "Toggle rhythmic sl&ash notation"),
-             TranslatableString("action", "Toggle ‘rhythmic slash notation’")
+             TranslatableString("action", "Toggle rhythmic slash notation")
              ),
     UiAction("pitch-spell",
              mu::context::UiCtxNotationOpened,
@@ -1858,13 +1852,13 @@ const UiActionList NotationUiActions::m_actions = {
     UiAction("toggle-mmrest",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_ANY,
-             TranslatableString("action", "Toggle 'Create multimeasure rest'"),
-             TranslatableString("action", "Toggle multimeasure rest")
+             TranslatableString("action", "Toggle multimeasure rests"),
+             TranslatableString("action", "Toggle multimeasure rests")
              ),
     UiAction("toggle-hide-empty",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_NOTATION_OPENED,
-             TranslatableString("action", "Toggle 'Hide empty staves'"),
+             TranslatableString("action", "Toggle empty staves"),
              TranslatableString("action", "Show/hide empty staves")
              ),
     UiAction("set-visible",
@@ -1882,14 +1876,14 @@ const UiActionList NotationUiActions::m_actions = {
     UiAction("toggle-autoplace",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_ANY,
-             TranslatableString("action", "Toggle 'automatic placement' for selected elements"),
+             TranslatableString("action", "Toggle automatic placement for selected elements"),
              TranslatableString("action", "Toggle automatic placement for selected elements")
              ),
     UiAction("autoplace-enabled",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_NOTATION_OPENED,
-             TranslatableString("action", "Toggle 'automatic placement' (whole score)"),
-             TranslatableString("action", "Toggle ‘automatic placement’ (whole score)")
+             TranslatableString("action", "Toggle automatic placement for entire score"),
+             TranslatableString("action", "Toggle automatic placement for entire score")
              ),
     UiAction("string-above",
              mu::context::UiCtxNotationOpened,
```
</details>

<details><summary><b>Note-input options:</b></summary>

```diff
diff --git a/src/notation/internal/notationuiactions.cpp b/src/notation/internal/notationuiactions.cpp
index b417f996a1..3562f94477 100644
--- a/src/notation/internal/notationuiactions.cpp
+++ b/src/notation/internal/notationuiactions.cpp
@@ -212,12 +212,6 @@ const UiActionList NotationUiActions::m_actions = {
              TranslatableString("action", "Previous system"),
              TranslatableString("action", "Go to previous system")
              ),
-    UiAction("toggle-insert-mode",
-             mu::context::UiCtxNotationOpened,
-             mu::context::CTX_ANY,
-             TranslatableString("action", "Toggle 'insert mode'"),
-             TranslatableString("action", "Note input: toggle ‘insert’ mode")
-             ),
     UiAction("select-next-chord",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_ANY,
@@ -1907,50 +1901,56 @@ const UiActionList NotationUiActions::m_actions = {
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_ANY,
              TranslatableString("action", "Note input"),
-             TranslatableString("action", "Note input: toggle note input mode"),
+             TranslatableString("action", "Note input: toggle on/off"),
              IconCode::Code::EDIT,
              Checkable::Yes
              ),
+    UiAction("toggle-insert-mode",
+             mu::context::UiCtxNotationOpened,
+             mu::context::CTX_ANY,
+             TranslatableString("action", "Toggle insert/overwrite mode"),
+             TranslatableString("action", "Note input: toggle mode: insert/overwrite")
+             ),
     UiAction("note-input-steptime",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_NOTATION_OPENED,
              TranslatableString("action", "Default (step time)"),
-             TranslatableString("action", "Note input: toggle ‘default (step-time)’ mode"),
+             TranslatableString("action", "Note input: toggle mode: default (step-time) / off"),
              IconCode::Code::EDIT
              ),
     UiAction("note-input-rhythm",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_NOTATION_OPENED,
              TranslatableString("action", "Rhythm only (not pitch)"),
-             TranslatableString("action", "Note input: toggle ‘rhythm only (not pitch)’ mode"),
+             TranslatableString("action", "Note input: toggle mode: rhythm only (not pitch) / off"),
              IconCode::Code::RHYTHM_ONLY
              ),
     UiAction("note-input-repitch",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_ANY,
              TranslatableString("action", "Re-pitch existing notes"),
-             TranslatableString("action", "Note input: toggle ‘re-pitch existing notes’ mode"),
+             TranslatableString("action", "Note input: toggle mode: re-pitch existing notes / off"),
              IconCode::Code::RE_PITCH
              ),
     UiAction("note-input-realtime-auto",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_NOTATION_OPENED,
              TranslatableString("action", "Real-time (metronome)"),
-             TranslatableString("action", "Note input: toggle ‘real-time (metronome)’ mode"),
+             TranslatableString("action", "Note input: toggle mode: real-time (metronome) / off"),
              IconCode::Code::METRONOME
              ),
     UiAction("note-input-realtime-manual",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_NOTATION_OPENED,
              TranslatableString("action", "Real-time (foot pedal)"),
-             TranslatableString("action", "Note input: toggle ‘real-time (foot pedal)’ mode"),
+             TranslatableString("action", "Note input: toggle mode: real-time (foot pedal) / off"),
              IconCode::Code::FOOT_PEDAL
              ),
     UiAction("note-input-timewise",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_NOTATION_OPENED,
              TranslatableString("action", "Insert"),
-             TranslatableString("action", "Note input: toggle ‘insert’ mode (increases measure duration)"),
+             TranslatableString("action", "Note input: toggle mode: insert (increases measure duration) / off"),
              IconCode::Code::NOTE_PLUS
              ),
     UiAction("realtime-advance",
```

</details>

I moved the `"toggle-insert-mode"` action down to join the other note input options.

Technically 'insert' is a verb, but I think `Toggle insert mode` works without quotes. Text editors are widely understood to have an insert mode and an overwrite mode.

**Comment by Gootector** [comment]
I changed the title and description and moved the approved changes here: https://github.com/musescore/MuseScore/pull/19888.

File: src/notation/internal/notationuiactions.cpp (hunk 1 above)
Comment (Jojo-Schmitz) [reply src/notation/internal/notationuiactions.cpp:218]: Wasn't the consensus to drop the quotes entirely?

**Comment by Gootector** [comment]
@Jojo-Schmitz I don't know 😆 I'm thinking... 🌀 Work in progress... Please wait...

**Comment by Gootector** [comment]
@Jojo-Schmitz I don't understand, why did shoogle add spaces around '/' mark?

**Comment by Gootector** [comment]
OK. Done.

**Comment by Gootector** [comment]
A big mess with /
Go to previous measure / move text left quickly
Select note/rest above
' / ' vs '/'

**Comment by shoogle** [comment]
> why did shoogle add spaces around '/' mark?

'/' means 'or'. When there are spaces around it that indicates there are more words involved, which helps to avoid ambiguity in some cases.

Take these sentences for example:

1. We sell coffee with milk/sugar.
2. We sell coffee with milk / sugar.

Here's what they mean according to my rule:

1. You can buy coffee with milk, or you can buy coffee with sugar.
2. You can buy coffee with milk, or you can buy sugar on its own (i.e. with no coffee)!

**Review by Jojo-Schmitz — APPROVED** [review]

**Comment by Gootector** [comment]
> > why did shoogle add spaces around '/' mark?
> 
> '/' means 'or'. When there are spaces around it that indicates there are more words involved, which helps to avoid ambiguity in some cases.
> 
> Take these sentences for example:
> 
> 1. We sell coffee with milk/sugar.
> 2. We sell coffee with milk / sugar.
> 
> Here's what they mean according to my rule:
> 
> 1. We sell coffee with (milk or sugar).
> 2. We sell (coffee with milk, or sugar).
> 
> In other words:
> 
> 1. You can buy coffee with milk, or you can buy coffee with sugar.
> 2. You can buy coffee with milk, or you can buy sugar on its own (i.e. with no coffee)!

In my horrible opinion, you drink too much coffee with milk and sugar. It's harmful to health. Things should be as simple as possible. See my changes.

**Review by shoogle — APPROVED** [review]

**Comment by Gootector** [comment]
In my horrible opinion is OK.

**Comment by shoogle** [comment]
GitHub Actions seems to be having problems today, so I'm going to merge this even though the checks haven't run yet. It only changes strings, and checks completed successfully on previous versions of the PR. If there are problems I will fix them in PR #19900, which is currently held up by this one.

**Comment by Gootector** [comment]
> GitHub Actions seems to be having problems today, so I'm going to merge this even though the checks haven't run yet. It only changes strings, and checks completed successfully on previous versions of the PR. If there are problems I will fix them in PR #19900, which is currently held up by this one.

@shoogle OK. Greetings to You and have a nice day!

**Pull request merged**